pywr/recorders/_hydropower.pyx

```diff
@@ -23,11 +23,11 @@ cpdef double hydropower_calculation(double flow, double water_elevation, double
         An efficiency scaling factor for the power output of the turbine.
     flow_unit_conversion : double (default=1.0)
         A factor used to transform the units of flow to be compatible with the equation here. This
-        should convert flow to units of $m^3/day$
+        should convert flow to units of $m^3/second$
     energy_unit_conversion : double (default=1e-6)
-        A factor used to transform the units of power. Defaults to 1e-6 to return $MJ$/day. 
+        A factor used to transform the units of power. Defaults to 1e-6 to return $MW$. 
     density : double (default=1000)
-        Density of water in $kgm^{-3}$.
+        Density of water in $kg/m^{-3}$.
```

Comment:
Density is a measure of unit mass per unit volume, so this should either be `kg / m^3` or `kg m^{-3}` as it was.

Reply:
Yes, I agree.
I will correct that